### ISSUE TITLE
No Differentiation of supporters and non-supporters in chat.

### ISSUE THREAD

**Issue by tomdenty1** [ppy/osu#2500]
![image](https://user-images.githubusercontent.com/36710174/39784337-f25610d2-530f-11e8-9bfd-ebbaa623b311.png)

Supporters and Non-Supporters are indistinguishable from each other in chat.

**Comment by peppy** [comment]
We do have a design for this, I believe.

**Comment by arflyte** [comment]
![d67d3b38cd](https://user-images.githubusercontent.com/3632750/39811041-6a398770-53c2-11e8-839c-c004592bacdd.png)

There's user group colours next to the next, which is not implemented yet.

**Comment by Tom94** [comment]
So, if user group colours are indicated by those little bars, will the names of people with a coloured forum name (such as peppy) be randomly coloured again, or are supporters vs. non-supporters separate from coloured forum names?

**Comment by tomdenty1** [comment]
Supporters being able to customise their name colour would be a pretty nice idea

**Comment by peppy** [comment]
Wait, so being a "supporter" makes you in a different group? That seems wrong and hard to parse.

**Comment by arflyte** [comment]
the bars are just indicators of usergroup. the name colours for supporter and non-supporters are random. special usergroups name colour will be the same as the bar's colour.

the bar might go through some slight redesign to match the other page designs.

**Comment by peppy** [comment]
This issue is about supporters specifically. How are they distinguished?

**Comment by arflyte** [comment]
I'll look into again.

**Comment by Sjshi763** [comment]
Maybe it can fit on the far right?

**Comment by Sjshi763** [comment]
Including these spaces on mobile, I think it's also sufficient
